lib/usage.ts

```diff
@@ -164,6 +164,9 @@ export function usage(yargs: YargsInstance, shim: PlatformShim) {
   };
 
   function getWrap() {
```

Comment:
So that we can maintain our coverage #s, I would do this:

```js
this.getWrap = () => {
  ...
}
```

And then your test can simply be:

```js
const yargs = yargs().wrap(99);
process.env.YARGS_DISABLE_WRAP = 'true';
expect(yargs.getInternalMethods().getUsageInstance().getWrap()).to.equal(null);
```

Reply:
I made the changes 👍